app/services/energy_tariff_default_prices_creator.rb

```diff
@@ -10,6 +10,7 @@ def process
 
     night_times = case @energy_tariff.tariff_holder_type
                   when 'School' then Economy7Times.times(@energy_tariff&.meters&.first&.mpan_mprn)
+                  when 'SchoolGroup' then TimeOfDay.new(0, 0)..TimeOfDay.new(7, 0) # Equivalent to Economy7Times::DEFAULT_TIMES
```

Comment:
Constant should be visible now in latest analytics. Could do this later as part of further improvements.

Reply:
:+1: